src/Header/Search/Search.jsx

```diff
@@ -125,7 +126,8 @@ Search.propTypes = {
 	]),
 	placeholder: PropTypes.string,
 	query: PropTypes.string,
-	onSearching: PropTypes.oneOfType([PropTypes.string, PropTypes.func])
+	onSearching: PropTypes.oneOfType([PropTypes.string, PropTypes.func]),
+	onNavigating: PropTypes.func
```

Comment:
This can be string or func like the onSearching prop:
```suggestion
	onNavigating: PropTypes.oneOfType([PropTypes.string, PropTypes.func])
```